frontend/src/components/templates/hooks/use-list-item-navigation.ts

```diff
@@ -36,7 +36,8 @@ export const useListItemNavigation = () => {
         });
       }
 
-      return navigate('/', { state: { message: successMessage } });
+      // @ts-ignore
```

Comment:
please don't add ts-ignore

Reply:
As building are own components `ts-ignore` is definitely a pattern to avoid but when using third-party librairies it can be hard to avoid. Here the types define from `react-router` V6 say that we can't pass a number to the `navigate` function, which is a pattern frequently used according to other developers in the community. Anyway, I'll try to find a better solution !